generators/kubernetes/templates/skaffold/skaffold.yml.ejs

```diff
@@ -0,0 +1,17 @@
+apiVersion: skaffold/v2alpha1
+kind: Config
+build:
+  artifacts:
+<%_
+appConfigs.forEach((appConfig) =>  {
```

Comment:
In this part we have weird behavior, it took me some moments to perfectly reconstruct the issue, and here it is:

You will get an error for microservices like this:



```
 create kustomization.yml
SyntaxError: Document is not valid YAML (bad indentation?) (11:3)
   9 |     envTemplate:
  10 |       template: "my-container-repo/gw:latest"
> 11 |   - image: my-container-repo/msvc
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 12 |     context: ../msvc
     | ^^^^^
  13 |     jib: {}
  14 |   tagPolicy:
  15 |     envTemplate:
    at t (/home/david/jhipster/generator-jhipster/node_modules/prettier/parser-yaml.js:1:341)
    at Object.parse (/home/david/jhipster/generator-jhipster/node_modules/prettier/parser-yaml.js:15:162854)
    at Object.parse (/home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:9739:19)
    at coreFormat (/home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:13252:23)
    at format (/home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:13510:73)
    at formatWithCursor (/home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:13526:12)
    at /home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:44207:15
    at Object.format (/home/david/jhipster/generator-jhipster/node_modules/prettier/index.js:44226:12)
    at prettier.resolveConfig.then.options (/home/david/jhipster/generator-jhipster/generators/generator-transforms.js:44:39)
```

Here is my JDL:

```
application {
    config {
        baseName uaa
        applicationType uaa
        cacheProvider no
        enableHibernateCache true
        buildTool gradle
    }
}

application {
    config {
        baseName msvc
        applicationType uaa
        cacheProvider no
        enableHibernateCache true
        buildTool gradle
    }
    
    entities *
}


application {
    config {
        baseName gw
        applicationType gateway
        authenticationType uaa
        uaaBaseName "UserService"
        devDatabaseType no
        prodDatabaseType no
        databaseType no
        buildTool gradle
    }
    
    entities *
}

entity Foo {
    name String required
}

entity Bar {
    name String required
}

microservice Foo, Bar with msvc
```

Note: if you use service names with upper-case characters, this error does not occur, but according to the line:

```
if(appConfig.baseName.toLowerCase() === appConfig.appFolder) {
```

Here I ask my self, if this is actual true....I've actually no idea what are your thoughts behind it....anyways, if your services names are lower-case AND are matching the appFolder, it happens

Reply:
I reverted the change now. Thnx. I added it for a different reason just to keep it to the cwd. Apparently the whole time when I was testing this feature, it ended up producing entries only from the cwd in spite of selecting multiple services. I guess it maybe because of this change, https://github.com/jhipster/generator-jhipster/blob/31a450e8332df14a3da834ac89369dfce3e83d71/generators/docker-base.js#L108

@sendilkumarn 

Let's say if I have 2 services like,
'micro/svc1'
'micro/svc2'

When I run the generator from micro/svc1 by selecting both the service (svc1 and svc2) options, it ends up producing two entries of svc1 instead of svc1 and svc2. Loop returns two entries of the svc (in context) from the current working directory (cwd). `false` bool argument to  getAllJhipsterConfig(... ) seems to be causing this anomaly. When I change it to true, it produces the expected o/p. 

If I run the generator from 'micro' folder, it works as expected even with `false`. I see this misbehaviour only when you run the generator from any of the services folder.

Reply:
// @sendilkumarn

Reply:
Oops sorry for the delayed response apparently the `notifications` didn't work properly :( 

It should be `true`. Without it this will take only the CWD configuration. I will send a patch today :)

Reply:
Done in https://github.com/jhipster/generator-jhipster/pull/11241 :)